packages/components/src/DateTimePickers/ViewLayout/ViewLayout.scss

```diff
@@ -0,0 +1,15 @@
+
+.container {
+    height: 300px;
```

Comment:
We need the same height between the two views to avoid the resize effect when switching.
I don't know if we need to fix it in the widget (by the ViewLayout or DateTimePicker component) or handle it externally (and so manage it relatively inside).

Reply:
Done in the main widget in rem
&#10003;